### ISSUE TITLE
Time Distributed dense layer vs dense layer accuracy difference

### ISSUE THREAD

**Issue by monikatomar92** [keras-team/keras#10502]
Hi

Find below snippets of code that were used to get accuracy results, while dense gave upwards of 50% accuracy for 100 epochs consistently, time distributed version of the model gave 22%, isn't time distributed dense and dense supposed to be the same beginning keras 2.0?

Code1: (Dense)

model = Sequential()
model.add(LSTM(RNN_units, name='rnn_0', return_sequences=True, input_shape=(300,360)))  
for i in range(rnn_layers):
	model.add(LSTM(RNN_units, name='rnn_' + str(i+1), return_sequences=True))  
model.add(SimpleRNN(50, name='rnn_'+ str(i+2), return_sequences=False))  
model.add(Dropout(0.5))
model.add(Dense(8, activation = 'softmax'))
model.summary()
model.compile(loss='categorical_crossentropy', optimizer='adam', metrics=['accuracy'])

Code2: (Time Distributed Dense)

model = Sequential()
model.add(LSTM(RNN_units', name='rnn_0', return_sequences=True, input_shape=(300,360)))  
for i in range(rnn_layers):
	model.add(LSTM(RNN_units, name='rnn_' + str(i+1), return_sequences=True))  
model.add(SimpleRNN(50, name='rnn_'+ str(i+2), return_sequences=True))  
model.add(Dropout(0.5))
model.add(TimeDistributed(Dense(8, activation = 'softmax')))
#model.add(Activation('sigmoid'))
model.summary()
model.compile(loss='categorical_crossentropy', optimizer='adam', metrics=['accuracy'])

**Comment by tRosenflanz** [comment]
They are not same. One returns sequences, another one doesn't - that's why your accuracy is going down since earlier predictions are not operating on all of the given data
